main.css

```diff
@@ -0,0 +1,228 @@
+body {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.container{
+  display: grid;
+  grid-template: auto 2fr 2fr 2fr 2fr 2fr 3.5fr 2fr auto / 2fr 1fr 1fr;
+  grid-template-areas: 
+  "n n n"
+  "t t t"
+  "q1 q1 q1"
+  "w w w"
+  "w w w"
+  "q2 q2 q2"
+  "team team team"
+  "s s s"
+  "f f f";
+}
+
+.nav {
+  grid-area: n; 
+  margin: 20px;
+  flex-wrap: wrap;
+}
+.nav ul {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  list-style-type: none;
+}
+
+.try {
+text-align: right;
+color: #B39662;
+padding-left: 380px;
+}
+
+.about {
+  padding-right: 80px;
+
+}
+
+.tagline {
```

Comment:
Your nav and tagline classes are designed so well and look great in the browser!